ui/pages/confirm-approve/confirm-approve.js

```diff
@@ -172,6 +172,7 @@ export default function ConfirmApprove({
           customTokenAmount={String(customPermissionAmount)}
           dappProposedTokenAmount={tokenAmount}
           currentTokenBalance={tokenBalance}
+          setApproveForAllArg={setApproveForAllArg}
```

Comment:
Nit: `setApproveForAllArg` doesn't seem meaningful in name; since it's only used for the warning at the moment, should we call it "showWarning" or something to that effect?

Reply:
Oh, I commented on this in the PR description, although vaguely. "Should be promptly followed by this (non-functional) code cleanup https://github.com/MetaMask/metamask-extension/issues/15805". I wanted to resolve that in another PR because it was pre-existing and used in other places.